dist/common_functions.py

```diff
@@ -0,0 +1,22 @@
+#!/bin/python3
+
+import subprocess
+
+# Common Python functions shared across dist/ scripts
+
+def last_commit_from_dev():
+    # Find the commit from develop at which point the current branch diverged.
+    # rev-list will show all commits that are present on our current branch but not on develop, and 
+    # the oldest of these is our first commit post-divergence. If this commit exists then 
+    # we can take its parent. If no such commits exist then we're currently on a commit in the 
+    # develop branch and can use HEAD instead
+
+    earliest_commit = subprocess.run( "git rev-list HEAD...develop | tail -n 1", 
+        shell=True, capture_output=True, text=True).stdout
+
+    if earliest_commit != "":
```

Comment:
I think this test is enough:

```suggestion
    if earliest_commit:
```

Reply:
Fixed in [c571e16](https://github.com/wiredtiger/wiredtiger/pull/9397/commits/c571e16c8e90a2e698247d4fd17ffc5ec664891a)